src/module-config/module-config.test.ts

```diff
@@ -34,4 +34,10 @@ describe("getConfig", () => {
       /schema.*defined/
     );
   });
+
+  it("receives config file from import map", () => {});
```

Comment:
Let's implement these tests and then merge this. We can follow up with more features in subsequent PRs

Reply:
Ok, I implemented these tests and wrapped the import map resolution code in a function.